blocks/library/heading/index.js

```diff
@@ -29,6 +29,8 @@ registerBlockType( 'core/heading', {
 
 	className: false,
 
+	supportAnchor: true,
```

Comment:
It was just an easy way to override the default generated className if a plugin author wanted to do so. But we could remove it for now.